location-awareness.md

```diff
@@ -89,3 +122,7 @@ In this case, PD will schedule different replicas of each datum to different dat
 - If the data zone cannot recover within a period of time, PD will remove the replica from this data zone.
 
 To sum up, PD maximizes the disaster recovery of the cluster according to the current topology. Therefore, if you want to reach a certain level of disaster recovery, deploy many machines in different sites according to the topology. The number of machines must be more than the number of `max-replicas`.
+
+However, with setting `isolation-level` to `zone`, PD will strictly ensure that each replica will isolated from each other at the zone level, even if guaranteeing this will not meet the number requirement of `max-replicas`. For example, there are three data zones z1/z2/z3 in the TiKV cluster. Under the requirement of 3 replicas, PD will dispatch the three replicas of the same Region to these three data zones. If there is a power outage in the z1 and it cannot be recovered after a period of time, PD will think that the region replica on z1 is no longer available, but because the `isolation-level` is set to `zone`, PD needs to strictly guarantee different region replicas will not fall on the same data zone. Because both z2 and z3 already have copies, PD will not perform any scheduling under the minimum isolation level limit of `isolation-level`, even if there are only two copies at this moment.
+
+Similarly, when `isolation-level` is `rack`, the minimum isolation level will be different racks in the same data zone. Under this configuration, the isolation at the zone level will be guaranteed first if it can be. When the isolation at the zone level cannot be promised, PD will consider to avoid scheduling replicas to be in the same zone with the same rack, and so on.
```

Comment:
```suggestion
Similarly, when `isolation-level` is set to `rack`, the minimum isolation level applies to different racks in the same data zone. With this configuration, the isolation at the zone level is guaranteed first if possible. When the isolation at the zone level cannot be guaranteed, PD tries to avoid scheduling different replicas to the same rack in the same zone, and so on.
```

Reply:
The following paragraph is missing in your translation:

`总的来说，PD 能够根据当前的拓扑结构使得集群容灾能力最大化。所以如果用户希望达到某个级别的容灾能力，就需要根据拓扑结构在对应级别提供多于副本数 (`max-replicas`) 的机器。同时 TiDB 也提供了诸如 `isolation-level` 这样的强制隔离级别设置，以便更灵活地根据场景来控制对数据的拓扑隔离级别。`